k8s.jsonnet

```
@@ -109,26 +113,26 @@
     // May be the same as cutter_creds_file.
     sheetsync_creds: import "./google_creds.json",
 
+    // Path to a file containing a twitch OAuth token to use when downloading streams.
+    // This is optional (null to omit) but may be helpful to bypass ads.
+    downloader_creds_file: null,
+
     // The URL to write to the sheet for edit links, with {} being replaced by the id
-    edit_url: "https://wubloader.example.com/thrimbletrimmer/edit.html?id={}",
+    edit_url: "http://thrimbletrimmer.codegunner.com/edit.html?id={}",
```

i've been trying to remove this old url from the codebase. the example.com one is fine.
```suggestion
    edit_url: "https://wubloader.example.com/thrimbletrimmer/edit.html?id={}",
```

---

I have done this.